client/src/components/DownloadStarted.js

```diff
@@ -134,7 +134,7 @@ export const DownloadStarted = ({
               </Box>
             </WarningText>
           )}
-          {additionalRestrictions && (
+          {!isSampleMetadata && additionalRestrictions && (
```

Comment:
```suggestion
          {additionalRestrictions && (
```

Additional restrictions should still be displayed here.

Reply:
See the commit at d9aa9e95fcbeef55ae961c684749f3d19cbf5bbe.